internal/transform/span_test.go

```diff
@@ -344,7 +345,11 @@ func TestTransform(t *testing.T) {
 	for _, tc := range testcases {
 		tc.before()
 		mapper := NewMapper(ctx, tc.input.Snapshot(), logrus.New())
-		lumigoSpan := mapper.Transform()
+		invocationStartedTimestamp := now.UnixMilli()
+		if tc.expect.LambdaType == "function" && strings.HasSuffix(tc.expect.ID, "_started") {
```

Comment:
I think the LambdaType is always function no?

Reply:
No, sometimes it's "http". It's the name from OTL span, not the Lumigo span.
![image](https://user-images.githubusercontent.com/49228804/163780481-00f8946e-acf4-4025-97b7-6938683a54e1.png)